packages/arb-bridge-peripherals/package.json

```diff
@@ -11,14 +12,18 @@
     "deploy:buddydeployer": "hardhat run scripts/deploy_buddy_deployer.ts --network kovan5",
     "deploy": "yarn deploy:tokenbridge",
     "gen:uml": "sol2uml ./contracts/tokenbridge/arbitrum,./contracts/tokenbridge/ethereum,./contracts/tokenbridge/libraries -o ./gatewayUML.svg",
-    "gen:uml:test": "sol2uml ./contracts/tokenbridge/arbitrum,./contracts/tokenbridge/test -o ./gatewayTestUML.svg"
+    "gen:uml:test": "sol2uml ./contracts/tokenbridge/arbitrum,./contracts/tokenbridge/test -o ./gatewayTestUML.svg",
+    "slither": "slither --hardhat-artifacts-directory build/contracts ."
+  },
+  "workspaces": {
+    "nohoist": [
+      "@openzeppelin/contracts",
+      "@openzeppelin/contracts-upgradeable"
+    ]
   },
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@nomiclabs/hardhat-waffle": "^2.0.0",
-    "@openzeppelin/contracts": "3.4.0",
-    "@openzeppelin/contracts-upgradeable": "3.4.0",
-    "@openzeppelin/upgrades-core": "^1.7.6",
```

Comment:
Seems like you accidentally removed `upgrades-core` which is used to check storage layouts before upgrades

Reply:
ah indeed